content/automate/ManageIQ/Transformation/TransformationHosts/ovirt_host.class/__methods__/rolecheck.rb

```diff
@@ -9,7 +9,7 @@ def initialize(handle = $evm)
             end
 
             def main
-              playbook = "/usr/share/doc/ovirt-ansible-v2v-conversion-host-1.0.0/examples/conversion_host_check.yml"
+              playbook = "/usr/share/doc/ovirt-ansible-v2v-conversion-host-1.2.0/examples/conversion_host_check.yml"
```

Comment:
@fdupont-redhat 
How do these playbooks get into the appliance, is there a seeding process for each ansible enabled appliance?

Reply:
@mkanoor
The playbooks are not on the appliance, but on the RHV Manager. They are provided by the `ovirt-ansible-v2v-conversion-host` package. So nothing to do on ManageIQ side.